src/services/textChanges.ts

```diff
@@ -339,6 +345,12 @@ namespace ts.textChanges {
             this.insertText(sourceFile, token.getStart(sourceFile), text);
         }
 
+        public insertCommentThenNewline(sourceFile: SourceFile, character: number, position: number, commentText: string): void {

```

Comment:
>character [](start = 64, length = 9)

"character" seems like a confusing name for this.  Perhaps something about indentation?